### PR TITLE
feat: add unpub_auth cli tool

### PR DESCRIPTION
**Pull request by talisk**

### CODE REVIEW

**Comment by ekasetiawans** [comment]
unpub_auth executable not installed after calling dart pub global activate --source path ./
looks like you forgot to add executable section in pubspec.yaml

**Comment by talisk** [comment]
> unpub_auth executable not installed after calling dart pub global activate --source path ./ looks like you forgot to add executable section in pubspec.yaml

@ekasetiawans  I'm sorry for that. I have fixed it. Please try it again.

**Comment by ekasetiawans** [comment]
@talisk cool thanks

File: unpub_auth/lib/unpub_auth.dart [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+/// It's from [dart-lang/pub/lib/src/oauth2.dart](https://github.com/dart-lang/pub/blob/400f21e9883ce6555b66d3ef82f0b732ba9b9fc8/lib/src/oauth2.dart#L21:L22)
+const _identifier = '818368855108-8grd2eg9tj9f38os6f1urbcvsq399u8n.apps.'
+    'googleusercontent.com';
+
+/// The pub client's OAuth2 secret.
+///
+/// This isn't actually meant to be kept a secret.
+///
+/// It's from [dart-lang/pub/lib/src/oauth2.dart](https://github.com/dart-lang/pub/blob/400f21e9883ce6555b66d3ef82f0b732ba9b9fc8/lib/src/oauth2.dart#L27)
+const _secret = 'SWeqj8seoJW0w7_CpEPFLX0K';
```
Comment (jonasfj) [inline unpub_auth/lib/unpub_auth.dart:20]: Please go register your own oauth2 flow, don't reuse the one for pub.dev

**Pull request merged**